src/kitovu/sync/syncing.py

```diff
@@ -83,15 +83,20 @@ def start(connection_name: str, connection_settings: ConnectionSettings) -> None
     plugin.disconnect()
 
 
-def _sync_subject(subject: typing.Dict[str, str],
+def _sync_subject(subject: utils.JsonSchemaType,
```

Comment:
Strictly speaking this is wrong because we only used `JsonSchemaType` for things which are actually a JSON schema - but I think I'll unify `JsonSchemaType` and `JsonType` in `moodle.py` into one.